cmd/coap/main.go

```diff
@@ -43,7 +81,8 @@ func main() {
 
 	go func() {
 		coapAddr := fmt.Sprintf(":%d", cfg.Port)
-		errs <- ca.Serve(coapAddr)
+		logger.Log("info", fmt.Sprintf("Start CoAP server at %s", coapAddr))
```

Comment:
Align the messages to other adapters, ex. https://github.com/mainflux/mainflux/blob/master/cmd/http/main.go#L78